app_dart/lib/src/request_handlers/check_flaky_builders.dart

```diff
@@ -119,6 +120,10 @@ class CheckFlakyBuilders extends ApiRequestHandler<Body> {
     final Map<String?, PullRequest> nameToExistingPRs = await getExistingPRs(gitHub, slug);
```

Comment:
You don't have to do this PR, but this shouldn't rely on a YamlMap of Target. It should be using the scheduler class to get the actual target. You might run into this if you add ignore_flakiness at the platform level.

Reply:
Agreed, I'll open a bug to sweep these changes.

Reply:
I filed https://github.com/flutter/flutter/issues/87529